src/View/pagination.html.twig

```diff
@@ -0,0 +1,58 @@
+<html>
+<head>
+  <title>Message Board</title>
+</head>
+<body>
+
+{% if lastPage > 1 %}
+
+    {# the number of first and last pages to be displayed #}
+    {% set extremePagesLimit = 3 %}
```

Comment:
By `extreme` do you mean the maximum amount of pages you display?

Reply:
It defines how many of the first and last pages are displayed. If you've got 20 pages in total, the pages 1, 2, 3, 18, 19, 20 will always be displayed. It's a bad name, I guess